src/UserForm.js

```diff
@@ -292,25 +275,38 @@ class UserForm extends React.Component {
     submitter();
   }
 
-  getActionMenu = ({ onToggle }) => {
-    const { onCancel } = this.props;
-    const handleClick = () => {
-      onCancel();
-      onToggle();
-    };
+  getPaneFooter() {
+    const {
+      pristine,
+      submitting,
+      invalid,
+      onCancel,
+      submitButtonStyle,
+      cancelButtonStyle,
+    } = this.props;
+
+    const disabled = pristine || submitting || invalid;
 
     return (
-      <Button
-        data-test-cancel-user-form-action
-        buttonStyle="dropdownItem"
-        onClick={handleClick}
-      >
-        <Icon icon="times-circle">
-          <FormattedMessage id="ui-users.cancel" />
-        </Icon>
-      </Button>
+      <PaneFooter>
+        <Button
+          data-test-user-form-cancel-button
+          buttonStyle={cancelButtonStyle}
+          onClick={onCancel}
+        >
+          {<FormattedMessage id="ui-users.cancel" />}
+        </Button>
+        <Button
+          data-test-user-form-submit-button
+          buttonStyle={submitButtonStyle}
```

Comment:
I suggest removing `submitButtonStyle` and `cancelButtonStyle` properties at all and use values which you have specified in default props for them.